doc/processes/release_process.md

```diff
@@ -0,0 +1,109 @@
+
+OpenJ9 binaries are built by the AdoptOpenJDK community.  
+
+An OpenJ9 release should:
+
+* Use a single code base to support the various JDK-levels.
+* Track the upstream OpenJDK releases.  
+* Use a consistent set of tags across OpenJ9 & extensions repos to 
+ensure that releases can be rebuilt or branched later.
+* OpenJ9 releases should be high quality and pass a common quality bar. 
+* Communicate the tags for a release to AdoptOpenJDK to ensure that
+binaries at the correct levels have been created. 
+* Clearly define the supported platforms
+* Should not regress performance from release to release.
+
+
+## Release cadence
+OpenJ9 should aim to release quarterly to match the OpenJDK long term 
+support (LTS) release cadence.  OpenJDK 8 is released quarterly and 
+other LTS releases will likely follow similar release cadences.  There 
+should be monthly milestone builds as the project builds towards the 
+next release.
+
+## Quality bar
+Work with the AdoptOpenJDK community to define a common quality bar for 
+any OpenJDK-based JDK.  The aim is to define a set of tests, both 
+functional and stress, and 3rd party applications that should pass for 
+every release.
+
+## Platform support
+Define the platforms that are supported by this release.  The set of 
+platforms will depend on the availability of hardware at OpenJ9 and 
+AdoptOpenJDK that can be used to validate the platform meets the quality 
+bar.  The code base may support more platforms than can be validated by 
+the quality.  Such platforms will not be officially included in the release.
+
+## Performance
+OpenJ9 should avoid regressing performance from release to release.
+This is harder to validate in the open project given the limitted 
+hardware resources available.  Work is being done at AdoptOpenJDK to 
+enable performance testing.
+
+# Release Process
+
+1. Create an issue to track the release and record the information required
+above, including the SHAs and tags.  This information will mostly be duplicated
+with the Eclipse release plan.
+1. Ask committers - via Slack and mailing list -  to refrain from committing any
+further changes until an "all clear" signal is broadcast.  Disable the promotion
+from the `openj9-omr` `master` branch to the `openj9` branch.
+1. Pick a candidate OpenJ9 level to mark as the initial release candidate.  This
+should be a level that has successfully passed the nightly testing builds at 
+AdoptOpenJDK.
+1. Tag the `openj9` & `openj9-omr` repos at the specified level with a tag of the
+following form: `openj9-#releaseNumber#RC#candidatenumber#`.  For the `0.8` 
+release, this would result in a `openj9-0.8RC1` tag.
+1. Tag the Extensions repos at the levels that passed the AdoptOpenJDK builds with
+the same tag.
+1. Sound the "all-clear". The OpenJ9 `master` branch and `openj9-omr` promotion 
+can be reopened for pull requests at this point.
+1. Rebuild the tagged levels and ensure they pass the quality bar defined by 
```

Comment:
I suggest jenkins jobs get created to do this part. Then they can be faithfully repeated whenever needed (particularly if we could be making changes on branches, which I know we don't want to do regularly, but I really doubt we'll be able to completely avoid it).